cdc/sink/causality_test.go

```diff
@@ -215,6 +213,6 @@ func (s *testCausalitySuite) TestGenKeys(c *check.C) {
 		sort.Slice(keys, func(i, j int) bool {
 			return bytes.Compare(keys[i], keys[j]) > 0
 		})
-		c.Assert(keys, check.DeepEquals, tc.expected)
+		require.Equal(t, keys, tc.expected)
```

Comment:
```suggestion
		require.Equal(t, c.expected, keys)
```

The old and new assertion functions have different signatures, so please make sure you use the correct order for the rest as well.

Reply:
ok